src/tests/commonTests.ts

```diff
@@ -85,12 +90,14 @@ export async function renders(
     display: string;
   }
 ): Promise<void> {
-  const page = await simplePageSetup(componentTagOrHTML);
-  const element = await page.find(getTag(componentTagOrHTML));
+  it(`renders ${componentTagOrHTML}`, async () => {
```

Comment:
Let's keep this as `renders`. Otherwise, it could get very verbose if `componentTagOrHTML` is complex markup.